tests/lib/rules/self-closing-comp.js

```diff
@@ -107,73 +107,84 @@ ruleTester.run('self-closing-comp', rule, {
   invalid: [
     {
       code: 'var contentContainer = <div className="content"></div>;',
+      output: 'var contentContainer = <div className="content"/>;',
```

Comment:
@ljharb, @pl12133 - This should also take the rule `jsx-space-before-closing` into consideration. No space could be valid if `jsx-space-before-closing` is set as `never`.